pom.xml

```diff
@@ -477,7 +477,7 @@
     <version.com.sun.grizzly>1.9.59</version.com.sun.grizzly>
     <version.org.codehaus.jettison>1.4.0</version.org.codehaus.jettison>
     <version.junit>4.11</version.junit>
-    <version.org.jboss.byteman>4.0.10</version.org.jboss.byteman>
+    <version.org.jboss.byteman>4.0.9</version.org.jboss.byteman>
```

Comment:
I realise you have not requested a review yet but is this line expected? https://issues.redhat.com/browse/JBTM-3226

Reply:
Well the 4.0.10 didn't seem to be available. I will withdraw this PR while I investigate.